README.md

```diff
@@ -11,12 +11,8 @@
 
 Bevy is a refreshingly simple data-driven game engine built in Rust. It is free and open-source forever!
 
-## WARNING
-
-Bevy is still in the early stages of development. Important features are missing. Documentation is sparse. A new version of Bevy containing breaking changes to the API is released [approximately once every 3 months](https://bevyengine.org/news/bevy-0-6/#the-train-release-schedule). We provide [migration guides](https://bevyengine.org/learn/migration-guides/), but we can't guarantee migrations will always be easy. Use only if you are willing to work in this environment.
-
-**MSRV:** Bevy relies heavily on improvements in the Rust language and compiler.
-As a result, the Minimum Supported Rust Version (MSRV) is generally close to "the latest stable release" of Rust.
```

Comment:
I think we should keep the text the same since it also indicates why we tend to stick to latest stable.

For the newlines I think you can do:
```md
> some text.
>
> some more text
```

Reply:
Thanks, I'll do it now!

Reply:
![image](https://github.com/user-attachments/assets/a135d3dc-223b-432a-ae87-ec80893f79f8)

Dont it!